vars/pipeline2ATX.groovy

```diff
@@ -368,7 +372,12 @@ def createTestStep(row, debug) {
         description.addAll(crawlRows(child, true))
         child = child.nextTreeSibling
     }
-    testStep.put("description", description.join(""))
+    def allowedSchemaMaxStringLength = 117
+    if(description.join("").isEmpty()) {
+        testStep.put("description", "")
+    } else {
+        testStep.put("description", description.join("").take(allowedSchemaMaxStringLength) + "...")
+    }
```

Comment:
We don't care about the content of the pipeline prints.
```
if(description) {
    testStep.put("description", description.join(";").take(allowedSchemaMaxStringLength) + "...")
} else {
   testStep.put("description", "")
}
```
And we should use a better separator (e.g. ";")

Reply:
resolved in [efd391f](https://github.com/tracetronic/jenkins-library/pull/23/commits/efd391f6ebccb315d0cfa6909cd4fbc588bb41d0)